tests/datasets/api_tests.py

```diff
@@ -849,7 +845,9 @@ def test_export_dataset(self):
                 expected_response = export_table
                 break
         ui_export = yaml.safe_load(rv.data.decode("utf-8"))
-        self.assertEqual(ui_export[0], expected_response)
+        # TODO re-enable flaky next line
```

Comment:
https://github.com/apache/incubator-superset/pull/10731/checks?check_run_id=1091887012

for reference:
```
=================================== FAILURES ===================================
______________________ TestDatasetApi.test_export_dataset ______________________

self = <tests.datasets.api_tests.TestDatasetApi testMethod=test_export_dataset>

```